docs/pages/includes/enterprise/ent-user-prereqs.mdx

```diff
@@ -0,0 +1,31 @@
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted">
+
+- Access to the machine hosting the Auth Service
+
+- Maintaining SAML/OIDC auth connectors via the desktop CLI requires enterprise `tctl` admin tool version >= (=teleport.version=),
```

Comment:
```suggestion
- Maintaining SAML/OIDC auth connectors via the desktop CLI requires the enterprise `tctl` admin tool version >= (=teleport.version=),
```

Reply:
It's not clear to me in this tab what the relationship is between this bullet and the previous one. Should the user have both access to the Auth Service host _and_ the Enterprise version of `tctl` installed locally?